x-pack/filebeat/module/crowdstrike/falcon/ingest/auth_activity_audit.yml

```
@@ -23,7 +23,7 @@ processors:
       type: string
       ignore_missing: true
       ignore_failure: true
-      if: ctx?.crowdstrike?.event?.UserIp != null && ctx?.crowdstrike?.event?.UserIp != ""
+      if: ctx?.crowdstrike?.event?.UserIp != null && ctx?.crowdstrike?.event?.UserIp != "" && ctx?.crowdstrike?.event?.UserIp != "N/A"
```

I think a cleaner fix for this would be to use the `ip` type for the conversion. This means all the conditionals can go away and the validation happens during processing.
```suggestion
      type: ip
      ignore_missing: true
      ignore_failure: true
```